### PR TITLE
Node 12

### PR DESCRIPTION
**Pull request by jakogut**
Upgrade to Node 12 and Debian Bullseye in worker/core container images

This is currently based on the jakogut/qemu branch for testing, but will be rebased to master before merging.

### CODE REVIEW

File: core/package.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
 	},
 	"dependencies": {
 		"ajv": "^6.5.0",
-		"ajv-semver": "git+https://github.com/balena-io-modules/ajv-semver.git#resin-semver",
+		"ajv-semver": "^1.0.4",
```
Comment (klutchell) [inline core/package.json:10]: We might want to use this instead:
https://github.com/balena-io-modules/balena-semver

**Comment by klutchell** [comment]
> This is currently based on the jakogut/qemu branch for testing, but will be rebased to master before merging.

Which commits are new for this PR then? Just the last 2-3?

File: core/package.json (hunk 1 above)
Comment (jakogut) [reply core/package.json:10]: @klutchell Looking at both of these modules, it appears that the ajv-semver module is not directly replaceable by balena-semver, if at all.

**Comment by jakogut** [comment]
@balena-ci rebase

**Pull request merged**